lib/Bindings/Python/OMModule.cpp

```diff
@@ -153,6 +169,10 @@ PythonValue List::getElement(intptr_t i) {
   return omEvaluatorValueToPythonValue(omEvaluatorListGetElement(value, i));
 }
 
+PythonValue Tuple::getElement(intptr_t i) {
+  return omEvaluatorValueToPythonValue(omEvaluatorTupleGetElement(value, i));
```

Comment:
Please add a bounds check as the inner method doesn't seem to have one.

Reply:
I think the index operator of SmallVector in
```
/// Get an element of the Tuple.
OMEvaluatorValue omEvaluatorTupleGetElement(OMEvaluatorValue evaluatorValue,
                                            intptr_t pos) {
  return wrap(cast<evaluator::TupleValue>(unwrap(evaluatorValue).get())
                  ->getElements()[pos]);
}
```
does a bounds check. Or is that only enabled in assertions-on builds?